core/include/mmcore/utility/Picking.h

```diff
@@ -112,12 +114,13 @@ class PickingBuffer {
 
     std::map<unsigned int, std::vector<Interaction>> available_interactions;
     ManipVector_t pending_manipulations;
+    bool enabled;
+
 #ifdef WITH_GL
     int prev_fbo = 0;
     glowl::FramebufferObject* fbo = nullptr;
-    glowl::GLSLProgram* fbo_shader = nullptr;
+    glowl::GLSLProgram* shader = nullptr;
 #endif
```

Comment:
Why not drop the * ? Use plain objects.

Reply:
There wont't be any OpenGL realted headers be available in the core even when WITH_GL is defined. So, the only option here is to use forward declared classes via raw pointers in the header.

Reply:
Edit: Managed pointers also allow incomplete types, see commit 0e88026